selection/src/commonMain/kotlin/EnterElement.kt

```diff
@@ -0,0 +1,21 @@
+package com.juul.krayon.selection
+
+import com.juul.krayon.element.Element
+import com.juul.krayon.kanvas.Kanvas
+
+public class EnterElement : Element() {
+
+    override val tag: String get() = "enter"
+
+    public var next: Element? = null
+
+    override fun <E : Element> appendChild(child: E): E =
+        checkNotNull(parent).insertBefore(child, next)
+
+    override fun <E : Element> insertBefore(child: E, reference: Element?): E =
+        checkNotNull(parent).insertBefore(child, reference)
+
+    override fun <PAINT, PATH> draw(canvas: Kanvas<PAINT, PATH>) {
+        error("$tag should not be present in the DOM.")
```

Comment:
Since this isn't really _the_ DOM, should we refer to it as something else to avoid confusion?

Maybe something like?:

```suggestion
        error("$tag should not be present in the element tree.")
```

...I'm also fine w/ DOM if you prefer that.